docs/code.md

```diff
@@ -31,4 +32,72 @@ MLAgility provides two main tools, the `benchit` CLI and benchmarking APIs. Inst
 1. `evaluate_script()` uses a profiler to discover the model instances in the script, and passes each into the `benchmark_model()` API, which is defined in [api/model_api.py](https://github.com/groq/mlagility/blob/main/src/mlagility/api/model_api.py).
 1. The `benchmark_model()` API prepares the model for benchmarking (e.g., exporting and optimizing an ONNX file), which creates an instance of a `*Model` class, where `*` can be CPU, GPU, etc. The `*Model` classes are defined in [api/ortmodel.py](https://github.com/groq/mlagility/blob/main/src/mlagility/api/ortmodel.py), [api/trtmodel.py](https://github.com/groq/mlagility/blob/main/src/mlagility/api/trtmodel.py), etc.
 1. The `*Model` classes provide a `.benchmark()` method that benchmarks the model on the device and returns an instance of the `MeasuredPerformance` class, which includes the performance statistics acquired during benchmarking.
-1. `benchmark_model()` and the `*Model` classes are built on top of the [GroqFlow](https://github.com/groq/groqflow) framework.
\ No newline at end of file
+1. `benchmark_model()` and the `*Model` classes are built using [`onnxflow.build_model()`](#model-build-tool)
+
+# Model Build Tool
+
+The `onnxflow` package implements the `onnxflow.build_model()` function, which is an automated toolchain for building PyTorch/Keras/Hummingbird models into optimized ONNX files.
```

Comment:
Nit: Should this be called model conversion? We are not building the model here. IMO building implies compiling the model.

Reply:
Hmm that's legacy language from GroqFlow. Technically onnxflow can help you do a build after you convert, but that isn't the default behavior (default is conversion only).

Reply:
I'm going to say its still a "build" because we do the ONNX optimization and fp16 stages. I think of conversion as just the export stage. Is that ok?